tensorflow/core/kernels/gpu_prim_helpers_test.cu.cc

```diff
@@ -271,6 +271,22 @@ TEST_F(GpuPrimHelpersTest, GpuRadixSort_WithNumBits) {
   test::ExpectTensorEqual<int32>(expected_indices_out, *GetOutput(1));
 }
 
+TEST_F(GpuPrimHelpersTest, GpuRadixSort_WithNumBitsZero) {
+  // Check that num_bits=0 is handled correctly.
+  MakeRadixSort(DT_INT32, DT_INT32, /*need_keys_out=*/true, /*num_bits=*/0);
+  AddInputFromArray<int32>(TensorShape({8}), {4, 2, 6, 7, 1, 3, 0, 5});  // keys
+  AddInputFromArray<int32>(TensorShape({0}), {});                        // inds
```

Comment:
Can you also test the case where indices_in is specified, since there's a small codepath for that case?

Reply:
Done